code/core/system/proxy-fh16/include/CANMessageDataStore.h

```diff
@@ -51,6 +51,7 @@ class CanMessageDataStore : public automotive::odcantools::MessageToCANDataStore
     virtual void add(odcore::data::Container &container);
 
    private:
+    void writeToCANDevice(automotive::GenericCANMessage);
```

Comment:
@fgiaimo Could you please change the signature to const automotive::GenericeCANMessage& to avoid unnecessary copying at runtime?